src/components/SlidingPane/SlidingPane.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import classNames from 'classnames';
+import ExitIcon from '@moda/icons/exit-16';
+import { Clickable } from '../Clickable';
+import { Text } from '../Text';
+
+import './SlidingPane.scss';
+import { Stack } from '../Stack';
+
+export type SlidingPaneProps = React.InputHTMLAttributes<HTMLDivElement> & {
+  children: React.ReactNode;
+  title: string;
+  visible: boolean;
+  className?: string;
```

Comment:
I think this is redundant, `React.InputHTMLAttributes<HTMLDivElement>` already has `className`.